src/plugins/intel_gpu/src/plugin/program_builder.cpp

```diff
@@ -244,6 +244,12 @@ std::vector<cldnn::input_info> ProgramBuilder::GetInputInfo(const std::shared_pt
                                           || ov::is_type<ov::op::v1::Split>(prevOp)
                                           || ov::is_type<ov::op::v1::VariadicSplit>(prevOp)
                                           || ov::is_type<ov::op::v4::LSTMCell>(prevOp);
+
+        // Custom op also need to maintain output port.
+        if (m_custom_layers.find(prevOp->get_type_name()) != m_custom_layers.end()) {
+            is_legacy_multiple_outputs = false;
+        }
```

Comment:
do we need this code here for the exception? Does this happen when custom op is connected to split/variadiasplit/lstemcell?